starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,52 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime=0;
+  this.intervalId=0;
+  
+}
 
-// }
 
-// Chronometer.prototype.startClick = function () {
+//cada 1000ms se ejecuta una suma de 1 a currentTime
+Chronometer.prototype.startClick = function () {
+    this.intervalId = setInterval(function() {
+      this.currentTime++;
+      this.setTime();
+    }, 1000);
 
-// };
+};
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
+Chronometer.prototype.setMinutes = function () {
+  return Math.floor(this.currentTime/60);
+};
 
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+Chronometer.prototype.setSeconds = function () {
+  return this.currentTime%60;
+};
 
-// Chronometer.prototype.setTime = function () {
+Chronometer.prototype.twoDigitsNumber = function (number) {
+  if(number<10) {
+    return "0" + number;
+  } else {
+    return number.toString();
+  }
+};
 
-// };
+Chronometer.prototype.setTime = function () {
+  this.minutes = this.twoDigitsNumber(this.setMinutes());
+  this.seconds = this.twoDigitsNumber(this.setSeconds());
+};
```

Comment:
Mejor no definir this.minutes y this.seconds como propiedades del prototype, en este caso está confundiendo al programa. Si por el contrario las haces como variables y luego las pasarlas como parámetros de la función printTime(minutes, seconds) para que la pinte en el DOM